src/core/task.ts

```diff
@@ -41,22 +44,26 @@ export function blocking(method: Metric): boolean {
     return (elapsed > config.longtask);
 }
 
-export function start(method: Metric): void {
+export function start(method: Metric, resume: boolean = false): void {
     tracker[method] = performance.now();
+    counter[method] = resume ? counter[method] + 1 : 0;
 }
 
-export function stop(method: Metric): void {
+export function stop(method: Metric, pause: boolean = false): void {
```

Comment:
Not used. Why?

Reply:
I left it for consistency to be used in future since I added "resume" to start. If you feel this is redundant, I can remove it for now and add it back when we need it for any reason.